lib/tasks/backfill_routing_numbers.rake

```diff
@@ -0,0 +1,8 @@
+namespace :routing_number do
+  desc "backfill routing numbers"
+  task backfill: :environment do
+    BankAccount.where(_routing_number: nil).find_each do |ba|
```

Comment:
This might be good to write in SQL for performance and to avoid unnecessarily updating `updated_at`. Not a blocker.

Reply:
this cannot work because Rails is decrypting!